src/AmsConfiguration.cpp

```diff
@@ -603,6 +670,128 @@ bool AmsConfiguration::loadConfig81(int address) {
 	address += readInt(address, &i);
 	setDebugLevel(i);
 
+	bool domo = false;
+	address += readBool(address, &domo);
+
+	ackWifiChange();
+
+	return true;
+}
+//
+// domoticz (based on 81)
+//
```

Comment:
Merge this with loadConfig82 which will be the configuration version for v1.3.0

Reply:
ok!

Reply:
done